main.js

```diff
@@ -13,11 +13,13 @@ let timeout;
 exports.health = health.check;
 
 exports.get = fromURL => {
-	if(useCache){
```

Comment:
i think there'll need to be a bit of tidy up to remove `useCache` and others from this repo